tests/functional/oauth_query_param_validation.js

```diff
@@ -40,9 +41,11 @@ define([
     return openPageWithQueryParams(context, queryParams, '#fxa-400-header');
   };
 
-  var testErrorInclude = function (expected) {
-    return testElementTextInclude('.error', expected);
-  };
+  var testErrorInclude = thenify(function (expected) {
+    return this.parent
+      .then(visibleByQSA('.error'))
+      .then(testElementTextInclude('.error', expected));
```

Comment:
> think we should push `visibleByQSA` into `testElementTextInclude` so it's available universally?

That sounds like a fine idea to me, assuming there are no occasions when we wouldn't want to call `visibleByQSA` first (I can't think of any).